cmd/bootkube/render.go

```diff
@@ -21,6 +21,7 @@ const (
 	apiOffset                    = 1
 	dnsOffset                    = 10
 	etcdOffset                   = 15
+	bootEtcdOffset               = 200
```

Comment:
For below, if we render the TPR, which points to the migration cluster, ahead of time, we don't know what the pod IP will be. So instead we put it behind a service, which allows us to reserve an IP address ahead of time.